server/api/game/constants.js

```diff
@@ -82,3 +82,6 @@ export const newGhostValues = [
   { x: 23, y: 1, direction: 'LEFT' },
   { x: 1, y: 23, direction: 'LEFT' },
   { x: 23, y: 23, direction: 'LEFT' }];
+
+export const getEnergizers = () => [{ x: 3, y: 3 },
```

Comment:
No need to this. Energizers already initialized by the `getBoard()`

Reply:
I need the coordinates to check for the PacManOnEnergizer logic. Read the code well.